hazelcast/src/main/java/com/hazelcast/internal/management/TimedMemberStateFactory.java

```diff
@@ -309,15 +309,9 @@ private void handleDurableExecutorService(MemberStateImpl memberState,
         memberState.setDurableExecutorsWithStats(executorsWithStats);
     }
 
-    private void handleMultiMap(MemberStateImpl memberState, Config config,
+    private void handleMultiMap(MemberStateImpl memberState,
                                 Map<String, LocalMultiMapStats> multiMaps) {
-        Set<String> mapsWithStats = createHashSet(multiMaps.size());
-        for (String name : multiMaps.keySet()) {
-            if (config.findMultiMapConfig(name).isStatisticsEnabled()) {
```

Comment:
I don't see a replacement for this filtering logic. It's important for MC since we don't want to receive all of the DSes there, but only ones with stats enabled.

cc @emre-aydin

Reply:
`MultiMapService#getStats` is already doing this filtering, i'm relying on it.

Reply:
We had plans to show all data structures in MC whether stats are enabled or not but we haven't done that yet. So yes, for the moment, we are expecting to receive only the data structures that have stats enabled.

Reply:
Good to know. I've checked other `StatisticsAwareService` implementers and they also seem to be doing the filtering. 

Resolving this comment. Thanks for checking.